src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('tbody');
```

Comment:
You should select the table with class 'dashboard' instead of 'tbody'. Replace 'tbody' with '.dashboard'.

Reply:
Not resolved. If you have any questions - feel free to ask them in `fe_chat`

Reply:
Yeo, but whether it is searching by ".dashboard", it isn't working correctly. I wrote about that yesterday.